doc.go

```diff
@@ -22,6 +22,21 @@ of sub-packages.
 All clients in sub-packages are configurable via client options. These options are
 described here: https://godoc.org/google.golang.org/api/option.
 
+## Endpoint
```

Comment:
nit: I think we should call this section Endpoint Override.

Reply:
Updated, thanks!